README.md

```diff
@@ -30,6 +30,14 @@ Tag | Description
 
 The installation manual is available [here](https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker).
 
+If you want to enable whiteboard, configure it and run Docker Compose as follows:
+
```

Comment:
Please skip this, let's add it to the handbook instead, where all the docs currently are: https://jitsi.github.io/handbook/docs/devops-guide/devops-guide-docker#quick-start